app/util/bitbucket/populate_db.sh

```diff
@@ -23,6 +23,11 @@ BITBUCKET_DB_PASS="Password1!"
 # BITBUCKET version variables
 SUPPORTED_BITBUCKET_VERSIONS=(6.10.0 7.0.0)
 BITBUCKET_VERSION=$(sudo su bitbucket -c "cat ${BITBUCKET_VERSION_FILE}")
+if [[ -z "$BITBUCKET_VERSION" ]]; then
+        echo The $BITBUCKET_VERSION_FILE file does not exists or emtpy. Please check if BITBUCKET_VERSION_FILE variable \
```

Comment:
**does not exist**
and pls add dot in the end of the message.

Reply:
done